src/Dropdown.directive.js

```diff
@@ -155,9 +158,20 @@ const openDropdown = ({
   onOpen?.();
 };
 
-const closeDropdown = ({ dropdown, arrow, backgroundMask }, { onClose, temporaryHideAllDropdownsRef }, keepListenersAttached) => {
+const closeDropdown = ({
+  dropdown,
+  arrow,
+  backgroundMask,
+}, {
+  onClose,
+  temporaryHideAllDropdownsRef,
+}, keepListenersAttached) => {
   const isDropdownOpen = dropdown.getAttribute('open') === 'true';
   if (!isDropdownOpen) { return; }
+  if (dropdown.onVieportChangeInterval) {
+    clearInterval(dropdown.onVieportChangeInterval);
```

Comment:
Did you mean `viewport`?
```suggestion
    clearInterval(dropdown.onViewportChangeInterval);
```

Reply:
yep, working on all platforms.